node/cmd/guardiand/node.go

```diff
@@ -1818,3 +1866,29 @@ func argsConsistent(args []string) bool {
 
 	return true
 }
+
+// Parse a list of chainIds from a comma separated string and validate that they have a Transfer Verifier implementation
+func parseTxVerifierChains(
+	// A comma-separated list of Wormhole ChainIDs.
+	input string,
+) ([]vaa.ChainID, error) {
+	if len(input) == 0 {
+		return nil, errors.New("could not parse input to parseTxVerifierChains: input empty")
```

Comment:
If we wanted to disable transfer verification on all chains, wouldn't the empty input be a valid case?

Reply:
In that case we would just not pass in the flag at all, similar to how someone might omit the flag that enables the governor or sets a coingecko key. I think having a CLI interface where `--txVerifierChains=''` is a valid input is a bit messy.

Reply:
I'm probably misunderstanding how these Go command line flags work, but if you don't provide the flag what does the value default to? My current understanding is it would default to the value you specify in the declaration, which is an empty string; am I greping that wrong?

Reply:
The type is `*string` (pointer to a string) so the default is `nil`, I think

Reply:
I did some more testing and patched the code. The latest commit does the following:

- if the CLI flag is an empty string, panic (via logger.Fatal)
- if it's absent, txVerifier is disabled
- if it contains a list of chain IDs, enable Tx Verifier for those chains